x-pack/plugins/snapshot_restore/__jest__/client_integration/policy_add.test.ts

```diff
@@ -241,36 +237,37 @@ describe('<PolicyAdd />', () => {
           await nextTick();
         });
 
-        const latestRequest = server.requests[server.requests.length - 1];
-
-        const expected = {
-          config: {},
-          isManagedPolicy: false,
-          name: POLICY_NAME,
-          repository: repository.name,
-          retention: {
-            expireAfterUnit: 'd', // default
-            expireAfterValue: Number(EXPIRE_AFTER_VALUE),
-            maxCount: Number(MAX_COUNT),
-            minCount: Number(MIN_COUNT),
-          },
-          schedule: DEFAULT_POLICY_SCHEDULE,
-          snapshotName: SNAPSHOT_NAME,
-        };
-
-        expect(JSON.parse(JSON.parse(latestRequest.requestBody).body)).toEqual(expected);
+        expect(httpSetup.post).toHaveBeenLastCalledWith(
+          `${API_BASE_PATH}policies`,
+          expect.objectContaining({
+            body: JSON.stringify({
```

Comment:
since object keys are compared as encoded strings, order of the object body matters